builder/builder-next/exporter/writer.go

```diff
@@ -3,14 +3,15 @@ package containerimage
 import (
 	"context"
 	"encoding/json"
-	"runtime"
 	"time"
 
+	"github.com/containerd/containerd/platforms"
 	"github.com/moby/buildkit/cache"
+	binfotypes "github.com/moby/buildkit/util/buildinfo/types"
 	"github.com/moby/buildkit/util/progress"
 	"github.com/moby/buildkit/util/system"
-	digest "github.com/opencontainers/go-digest"
-	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/opencontainers/go-digest"
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
Probably good to keep this as `ocispec` (for now), as that's the alias we use elsewhere in the codebase (keeping it consistent makes it easier to find occurrences / uses of the same things). Also keep the `digest "github.com/opencontainers/go-digest"` (for now), I think there was some linter that preferred it, but if we remove it we should remove it everywhere (probably best to keep that separate from this PR)

Reply:
Sure